test/selenium/specs/tc_show_injected.rb

```diff
@@ -31,6 +31,19 @@ def test_showing_injected_posts
     end
   end
 
+  def test_showing_injected_posts_reddit
+
+    if not @@privly_extension_active
+      return
+    end
+
+    # Open these pages
+    test_page = "http://www.reddit.com/r/Privly/comments/2xipzl/privly_demonstration_link/"
+
+    visit test_page
+    assert page.has_css?(".usertext-body p iframe")
```

Comment:
Referencing the css of the page in this way is very brittle. I recommend finding the URL's string and looking for a sibling iframe. This will work even if they change the whole application.